grizzly/testdata/variables/messagequeue.py

```diff
@@ -49,6 +49,16 @@
 Given a user of type "RestApi" load testing "http://example.com"
 ...
 Then get request "fetch-document" from "/api/v1/document/{{ AtomicMessageQueue.document_id }}"
+
+# Using expression to get specific document by text value of "DocumentReference"
```

Comment:
Using one level  headers in pydoc-markdown will mess up the content tree in mkdocs. This should be a level 3 header.
Also, shorten it to "Using expression to get specific message".